### ISSUE TITLE
This code can't run sucessfully and Will the training code for Imagenet of Equal-Bits-BNN be released ?

### ISSUE THREAD

**Issue by lixcli** [liyunqianggyn/Equal-Bits-BNN#1]
Hi, I found some issues in this code.
1.  in `utils.py`:
```python
# line 38 to 40 should be:
        if parser_args.cuda:
            B_creat = B_creat.cuda()
            allones = allones.cuda()
```
2. in `resnet.py`:
```python
# line 26
self.rescale.data = torch.ones_like(self.weight.data) * std
# this function return a cpu tensor in default, so we need to control the type of self.rescale(cuda or cpu) manually.
# add this code below
if parser_args.cuda:
    self.rescale = self.rescale.cuda()
```
It can run sucessfully now. 
Will the training code for Imagenet of Equal-Bits-BNN be released ?

**Comment by liyunqianggyn** [comment]
Hi, thanks for informing:), I've updated the code. The ImageNet exp code will be released soon.

**Comment by liyunqianggyn** [comment]
ImageNet code is released, I will keep releasing the code of other architectures:  Conv2, Conv4, Conv6, Conv8, VGG16, InceptionV3 and ShuffleNet. Thanks for your interests. I will temporary close this question, you could reopen it whenever you have follow up question :)

**Comment by lixcli** [comment]
Thanks :)
